nuxt.config.js

```diff
@@ -1,7 +1,7 @@
 export default {
   // Global page headers: https://go.nuxtjs.dev/config-head
   head: {
-    title: 'authorization-bot',
+    title: 'NFT 4 Cause Discord Verification',
     htmlAttrs: {
```

Comment:
Let's keep this generic

Reply:
yep, that slipped into the PR. Need to keep that on a branch out of `main`.  My fault!